.codecov.yml

```diff
@@ -2,7 +2,7 @@ coverage:
   status:
     project:
       default:
-        target: 75%
+        target: 1%
```

Comment:
Is the target supposed to be this low? If it is, might as well remove the target completely. We can always leave the target higher and work towards it, we don't set a codecov pass as merge requirement.